doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
Hmmm.  I've wandered off the path then.  I thought the validators.txt was how a rippled instance selected which validators it wants to trust.  Do we now assume that all validators are trustworthy?  Which cog did I slip?